apis/datadoghq/v2alpha1/datadogagent_types.go

```diff
@@ -118,6 +118,37 @@ type APMFeatureConfig struct {
 	// Path Default: `/var/run/datadog/apm.socket`
 	// +optional
 	UnixDomainSocketConfig *UnixDomainSocketConfig `json:"unixDomainSocketConfig,omitempty"`
+
+	// Single Step Instrumentation allows the agent to inject the Datadog APM libraries into all pods in the cluster.
```

Comment:
```suggestion
	// SingleStepInstrumentation allows the agent to inject Datadog APM libraries into all pods in the cluster.
```
nit: since we usually put the verbatim parameter name as the first word in the comment